recipes/paraview/5.13.2/gh200/packages.yaml

```diff
@@ -0,0 +1,11 @@
+packages:
+  all:
+    providers:
+      gl: [egl]
+  egl:
+    externals:
+    - spec: egl@1.0.0
+      prefix: /usr
+    buildable: false
+  llvm:
+    require: llvm ~gold ~cuda
```

Comment:
Is this not already set in the environment for the `llvm` spec? Or does it concretize wrong without this?